hack/docker.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Script that builds and pushes docker images for multiple architectures. If you want to include a new architecture,
+# ensure that it is compiled in the build.sh script, then add your OS/ARCH combination to the PLATFORMS variable.
+
+PLATFORMS=linux/arm/v7,linux/arm64/v8,linux/amd64
+DOCKER_IMAGE=quay.io/jetstack/preflight
+
+# Uses the git tag or commit SHA.
+VERSION="$(bash ./hack/version.sh)"
```

Comment:
How does this work with tags? Does it run twice one with `VERSION=<commit sha>` and one with `VERSION=<tag>`?

Reply:
So when a tag is pushed, the `release-master` workflow will run, that `version.sh` script will then return the git tag instead of the SHA. Likewise, when `master` is pushed, the workflow will run, but since there's no tag it will return the SHA instead.

Reply:
Right! Makes sense.